backend/src/game/dice.py

```diff
@@ -0,0 +1,28 @@
+"""
+This is a dice module for the game King of Tokyo
+
+The dice can have the following face values:
+    - 1 - 3: These are the amount of victory points earned
+    - attack: Depending on the player's location, hurts either monsters
+    in or outside Tokyo
+    - heal: Heals the player up to a maximum of 10 points
+    - energy: Gives the player an energy cube to spend on upgrade cards
+"""
+
+import random
+
+DICE_VALUES = ["1", "2", "3", "attack", "heal", "energy"]
```

Comment:
I understand why you set everything to a string type, that way it is consistent; however, thinking about how the game will have to evaluate the dice value (eg. if there are 4 "2"s, you will have to cast that string into an int to understand what value it is but you only need to do that for integers, your function would not need to do this for attack, heal, and energy.  Since this is the case it makes sense for the integer values to remain as ints that may it is an easier thing to keep track of later on.  It may make sense to go even a step further and declare types for each of these dices

Reply:
If I used an enum however as the professor is suggesting, then I am going to have to do some type coercion still:

When we get to doing actual game coding, it's going to be just as easy to do something like:
``` Python
current_player.heal(sum([1 if die == "heart" else 0 for die in rolls]))
```
As it would to do:
``` Python

class FaceValues(enum):
    One = 0
    Two = 1
    Three = 2
    Attack = 3
    Heart = 4
    Energy = 5

current_player.get_victory_points(sum([int(die) + 1 if die in [FaceValues.One, FaceValues.Two, FaceValues.Three] else 0 for die in rolls]))
```

Reply:
Apparently when I reply by email, the reply doesn't go to the conversation. It just gets dumped at the bottom of the review lol. Check out below

Reply:
I don't like Face Values as I don't think a new dev would think to look for that name not something like Die or DieFaceValues

Reply:
Personally, I like the enum version.